copc/lidar-las-to-copc.ipynb

```diff
@@ -0,0 +1,707 @@
+{
```

Comment:
<p>It sounds like current versions of PDAL have just the pdal application and use the word "operations" to describe the different CLI commands.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>